### ISSUE TITLE
🔗 🚀 Feature Request: Ability to loop the entrance animation in Elementor (Merged with #30214)

### ISSUE THREAD

**Issue by ckcelite** [elementor/elementor#11114]
## Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro. )

**What problem is your feature request going to solve? Please describe.**
It would be great to be able to loop the entrance animation on Elementor objects.
**Describe the solution you'd like**
As an example, set the bounce on an image to happen 6 times, or indefinite times on page load.

![Elementor animation loop](https://user-images.githubusercontent.com/38842028/79050324-e4d7c400-7bee-11ea-804d-6d932fc7949f.jpg)

**Comment by arturbonnett** [comment]
:+1:

**Comment by sheidemann** [comment]
👍

**Comment by LeoBoudet** [comment]
+1

**Comment by jpaezsa** [comment]
👍

**Comment by lmabee** [comment]
+1

**Comment by RewForeN** [comment]
+1

**Comment by eSilverStrike** [comment]
+1 Another feature that really makes sense to add.

**Comment by warudin** [comment]
+1

**Comment by ckcelite** [comment]
Thank you gferenca, I will give this a try!

**Comment by eSilverStrike** [comment]
Elementor is adding more Animation features into the page builder (they are in the current beta) ... Why not this simple request?

**Comment by ckcelite** [comment]
> Elementor is adding more Animation features into the page builder (they are in the current beta) ... Why not this simple request?

I agree, seems like a simple addition and it would be very useful. Much better than using animated gifs.

**Comment by ckcelite** [comment]
> +1 Another feature that really makes sense to add.

Thanks! I hope they will.

**Comment by tekonemedia** [comment]
Yess we need this.

**Comment by mattpramschufer** [comment]
I know this thread is old, but you can simply use the Custom CSS when using Elementor Pro and then just use

`selector {
    animation-iteration-count: infinite!important;
}`

**Comment by mariobarretta** [comment]
Thankyou @mattpramschufer this has solved my problem.
Really strange that Elementor does not add this feature...

**Comment by shawnlee-info** [comment]
> I know this thread is old, but you can simply use the Custom CSS when using Elementor Pro and then just use
> 
> `selector { animation-iteration-count: infinite!important; }`

Who do you think didn't know doing that by code here? that's why people are asking Elementor to make it a codeless feature

**Comment by Daniyal857** [comment]
+1

**Comment by kevnoutsawo** [comment]
> this little css code betweeen the brackets did the trick . First part is my elementor image class so please put your element instead: `.elementor-641 .elementor-element.elementor-element-1462fc0 { animation-iteration-count: infinite!important; }`

Awesome solution. It works like a charm. Thanks a lot.
